core/store/src/main/java/uk/gov/gchq/gaffer/store/schema/SchemaElementDefinition.java

```diff
@@ -196,7 +197,7 @@ public ElementAggregator getOriginalAggregator() {
     }
 
     @JsonIgnore
-    public ElementAggregator getFullAggregator() {
+    public synchronized ElementAggregator getFullAggregator() {
```

Comment:
If the `IS2_INCONSISTENT_SYNC` is a false positive, does this need to have `synchronized` added?

Reply:
Good question, I was hoping that adding that would fix it, but it doesn't seem to have done. I'm not sure why this class has any `synchronized ` at all?